app/views/warning_bar/_protocol_mismatch.erb

```diff
@@ -0,0 +1,12 @@
+<div id="pending-migrations-warning" class="warning-bar--item unsupported-browser-warning--pane">
+  <span class="icon3 icon-warning"></span>
+  <p>
+    <strong>Protocol setting mismatch</strong>
+    <br/>
+    Your application is running with Protocol set to <code><%= Setting.protocol %></code>, but the request
+    is a <code><%= request.ssl? ? 'https' : 'http' %></code>  request.
```

Comment:
typo: a -> an (I think). Although 'h' is not a vowel, it is pronounced as such, almost like an 'a', in the context of 'http/s'.

The whole partial is not i18ned

Reply:
I actually wanted to skip I18n here because it's admin stuff anyway, but fine for me to translate it